client/src/db.js

```diff
@@ -0,0 +1,40 @@
+const allProjects = [
+    {
+        ID: 1,
```

Comment:
I'd suggest to use lowercase "id" here, because otherwise, you probably at some point will have the issue that you use different spellings here and somewhere else in your app; this would consume quite some time to find the bug ;)